typings/node/node.d.ts

```diff
@@ -0,0 +1,1335 @@
+// Type definitions for Node.js v0.10.1
+// Project: http://nodejs.org/
+// Definitions by: Microsoft TypeScript <http://typescriptlang.org>, DefinitelyTyped <https://github.com/borisyankov/DefinitelyTyped>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/************************************************
+*                                               *
+*               Node.js v0.10.1 API             *
```

Comment:
This is using the old v0.10.1 definitions. This should be updated to the newer v0.12.0 definitions: https://github.com/borisyankov/DefinitelyTyped/blob/master/node/node.d.ts